dss-core/src/main/java/io/github/ztkmkoo/dss/core/actor/DssServiceSpawnable.java

```diff
@@ -0,0 +1,71 @@
+package io.github.ztkmkoo.dss.core.actor;
+
+import akka.actor.typed.ActorRef;
+import io.github.ztkmkoo.dss.core.actor.exception.DssSpawnActorException;
+import io.github.ztkmkoo.dss.core.actor.property.DssMasterActorProperty;
+import io.github.ztkmkoo.dss.core.actor.property.DssServiceActorProperty;
+import io.github.ztkmkoo.dss.core.actor.util.DssCommonActorUtils;
+import io.github.ztkmkoo.dss.core.message.DssMasterCommand;
+import io.github.ztkmkoo.dss.core.message.DssServiceCommand;
+import io.github.ztkmkoo.dss.core.util.StringUtils;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * @author Kebron ztkmkoo@gmail.com
+ * @create 2020-09-15 03:04
+ */
+public interface DssServiceSpawnable extends DssServiceAcceptable {
+
+    /**
+     * Help create service actor behavior
+     */
+    DssBehaviorCreator<DssServiceCommand, DssServiceActorProperty> getServiceBehaviorCreator();
+
+    /**
+     * Create DssServiceActorProperty list from DssMasterActorProperty
+     */
+    <M extends DssMasterActorProperty> List<DssServiceActorProperty> createDssServiceActorPropertyList(M masterProperty);
+
+    /**
+     * Create DssServiceActorResolvable from DssServiceActorProperty and service actor
+     */
+    <P extends DssServiceActorProperty> DssServiceActorResolvable<String> createDssServiceActorResolvable(P property, ActorRef<DssServiceCommand> actor);
+
+    /**
+     * spawn service actors and set it
+     */
+    default <M extends DssMasterActorProperty> void initializeServiceActor(AbstractDssActor<DssMasterCommand> master, M masterProperty) {
+        final List<DssServiceActorProperty> propertyList = createDssServiceActorPropertyList(masterProperty);
+        if (Objects.isNull(propertyList) || propertyList.isEmpty()) {
+            return;
+        }
```

Comment:
Hi, I just wonder that propertyList.isEmpty() is necessary. 
I think createDssServiceActorPropertyList can return null when list is empty.
Or is this empty propertyList used in other parts?

Reply:
I think that it consider this case.
`createDssServiceActorPropertyList` can return null or empty list.
For example 
```java
final List<Integer> list = new ArrayList<>();

if (list.isEmpty()) {
    // list is empty and not null
}
if (Objects.isNull(list)) { }
```

```java
final List<Integer> list =null;

if (Objects.isNull(list)) {
    // list is null
}
/*
if (list.isEmpty()) { }
*/
```

Reply:
@RulLu16 
Hi there,
I check null and empty for @Doyuni pointed out.
(I did it automatically..)

But, as you said, it is not neccessary to check empty, if using Collection stream.
I will modify this